src/model/database.cpp

```diff
@@ -228,6 +228,19 @@ int ObjectDatabase::CountChildrenWithSameType(PObjectBase parent,PObjectType typ
 	return count;
 }
 
+int ObjectDatabase::CountChildrenWithTypes(PObjectBase parent,const std::set<wxString> &types)
+{
+	unsigned int count = 0;
+	unsigned int numChildren = parent->GetChildCount();
+	for (unsigned int i=0; i < numChildren ; i++)
```

Comment:
Missing whitespace around the equals sign, extra whitespace in front of the second semicolon.

Please don't use post-increment, always use pre-increment unless the use of post-increment is required